runtime/bcutil/attrlookup.gperf

```diff
@@ -24,9 +24,6 @@
  * This file is the input to the gperf utility for generating a perfect
  * hash table of attribute names, which is used by attributeTagFor().
  *
- * Types generated by gperf cause compiler issues in the OpenJ9 Windows builds,
- * use the existing types.
- *
```

Comment:
The header file `attrlookup.h` should be regenerated to reflect this change.

Reply:
Updated.